test/gravity/gravity_force_from_mass_test.py

```diff
@@ -0,0 +1,65 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+
+from symplyphysics import (
+    units, convert_to, SI, errors
+)
+from symplyphysics.laws.gravity import gravity_force_from_mass as gravity_law
+
+@fixture
+def test_args():
+    m1 = units.Quantity('m1')
+    SI.set_quantity_dimension(m1, units.mass)
+    SI.set_quantity_scale_factor(m1, 20000 * units.kilogram)
+    m2 = units.Quantity('m2')
+    SI.set_quantity_dimension(m2, units.mass)
+    SI.set_quantity_scale_factor(m2, 10000 * units.kilogram)
+    R = units.Quantity('R')
+    SI.set_quantity_dimension(R, units.length)
+    SI.set_quantity_scale_factor(R, 1 * units.meter)
+
+    Args = namedtuple('Args', ['m1', 'm2', 'R'])
+    return Args(m1 = m1, m2 = m2, R = R)
+
+def test_basic_force(test_args):
+    result = gravity_law.calculate_force(test_args.m1, test_args.m2, test_args.R)
+    assert SI.get_dimension_system().equivalent_dims(result.dimension, units.force)
+
+    result_force = convert_to(result, units.newton).subs(units.newton, 1).evalf(7)
+    assert result_force == approx(0.0133486, 0.0000001)
```

Comment:
Add high level description of what you are expecting to get. Preferrable to use some numbers from the internet because it looks like you calculated the result and compare with it. If the formula is incorrect you won't find an error this way.